src/main.h

```diff
@@ -77,17 +77,20 @@ static const int MAX_SCRIPTCHECK_THREADS = 16;
 /** -par default (number of script-checking threads, 0 = auto) */
 static const int DEFAULT_SCRIPTCHECK_THREADS = 0;
 /** Number of blocks that can be requested at any given time from a single peer. */
-static const int MAX_BLOCKS_IN_TRANSIT_PER_PEER = 16;
+//static const int MAX_BLOCKS_IN_TRANSIT_PER_PEER = 16;
+static int MAX_BLOCKS_IN_TRANSIT_PER_PEER = 1; // BU request manager just needs one per peer
```

Comment:
Comment is misleading, you are adjusting this dynamically based on block DL perf, right?  If so I will fix during merge.